crates/rolldown/tests/esbuild/ts/export_type_issue379/artifacts.snap

```diff
@@ -27,46 +28,3 @@ source: crates/rolldown_testing/src/integration_test.rs
 ───╯
 
 ```
-
```

Comment:
This should also reduce some snapshot noise due to there's no need to inspect bundle output of errored.